src/interface/get-order.interface.ts

```diff
@@ -0,0 +1,42 @@
+import { ICustomer } from '../interface';
+import { RetrieveInfo } from '../model';
+
+export interface IPayload {
+    customer: ICustomer;
+    info: RetrieveInfo;
+    items: IRetrieveItem[];
+    configuration: Record<string, any>;
+}
+
+export interface IRetrieveItem {
+    id: string;
+    'menu-item-name': string;
+    quantity: string;
+    price: string;
+    'special-instructions': string;
+    'menu-choices': string;
+    'menu-choices-hash': IMenuChoice;
+    'customer-name': string | null;
```

Comment:
maybe `'customer-name'?: string`?

Reply:
payload is schema is string, but in postman is object and special-instructions when response is string , i dont know if is optional.